src/services/VestingService.ts

```diff
@@ -18,9 +22,44 @@ export class VestingService {
   }
 
   static async getVestings(addresses: string[]): Promise<VestingWithLogs[]> {
-    const vestings = await Promise.all(addresses.map((address) => getVestingWithLogs(address)))
+    const vestingsData = await VestingsSubgraph.get().getVestings(addresses)
+    return vestingsData.map(this.parseSubgraphVesting).sort(compareVestingInfo)
```

Comment:
Unrelated but I think we should rename `compareVestingInfo` to something like `sortVestingsByDate`.